opentech/apply/funds/workflow.py

```diff
@@ -1,244 +1,47 @@
 from collections import defaultdict
-import copy
 import itertools
 
-from typing import Dict, Iterable, Iterator, List, Sequence, Set, Type, Union, TYPE_CHECKING
 
-from django.utils.text import slugify
-
-if TYPE_CHECKING:
-    from opentech.apply.users.models import User  # NOQA
-
-
-"""
```

Comment:
Can you keep the documentation?

The changes do read better and are easier to follow, but context is king!